app/models/option.rb

```diff
@@ -1,6 +1,10 @@
 class Option < ActiveRecord::Base
-  belongs_to :question
+  belongs_to :question, inverse_of: :options
   belongs_to :company
-  
-  validates :detail, presence: true, length: { maximum: 100 }
+
+  validates :detail, presence: true, length: { maximum: 100 }, if: :multiple_options?
```

Comment:
if this is string type then change length to 255 and if text type then change maximum length to 500